src/services/ui/src/features/package-actions/lib/index.ts

```diff
@@ -54,10 +55,10 @@ export const submitActionForm = async ({
     await submit({
       data: { ...data, id: id },
       endpoint: !actionsThatUseSubmitEndpoint.includes(type)
-        ? buildActionUrl(type!) // "/action/{type}"
+        ? buildActionUrl(type) // "/action/{type}"
         : "/submit",
       user,
-      authority: authority,
+      authority: AUTHORITY[authority],
```

Comment:
Does the authority need to be converted to lowercase before we send it to the API?